components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/constants/SQLConstants.java

```diff
@@ -3403,6 +3403,9 @@
                 "INSERT INTO AM_SYSTEM_APPS " + "(NAME,CONSUMER_KEY,CONSUMER_SECRET,TENANT_DOMAIN,CREATED_TIME) " +
                         "VALUES (?,?,?,?,?)";
 
+        public static final String GET__APPLICATIONS =
```

Comment:
Remove the concatenation and keep in a single line.

        `public static final String GET__APPLICATIONS = "SELECT * FROM AM_SYSTEM_APPS WHERE TENANT_DOMAIN = ?";`

Reply:
Ther are two _ characters in the variable.

Reply:
resolved